frontend/apps/pwa/package.json

```diff
@@ -29,6 +29,7 @@
     "core-api-client": "*",
     "core-auth": "*",
     "date-fns": "^2.25.0",
+    "lodash": "^4.17.21",
```

Comment:
I'm not sure adding lodash is a good idea. It seems to have been used for `map` and `forEach` which have no difference to the native versions. If there was something we wanted lodash for we should install the specific package (eg. `lodash.map`) as lodash is quite big.